formatters/swiftcov/Calculator.swift.gcov

```diff
@@ -0,0 +1,64 @@
+    -:    0:Source:/Users/katsumi/Dropbox/Xcode Projects/SwiftCov/Examples/ExampleFramework/ExampleFramework/Calculator.swift
```

Comment:
_checking again_
All the other formatters do have example files side by side, but in my working copy I have actually moved this into an examples directory.